src/task/script.js

```diff
@@ -303,3 +303,94 @@ function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
 }
 
 document.getElementById('visibity-hidden').classList.remove('visibity-hidden');
+
+const assigneeEl = document.getElementById('assignee');
+
+function debounce(func, delay) {
+  let searchDelay;
+  return (...args) => {
+    if (searchDelay) {
+      clearTimeout(searchDelay);
+    }
+
+    searchDelay = setTimeout(() => {
+      func(...args);
+    }, delay);
+  };
+}
+
+async function autoComplete(searchInput) {
+  try {
+    const response = await fetch(`${API_BASE_URL}/members`);
+    const data = await response.json();
+    removeAssignee();
+    if (searchInput !== '') {
+      const matches = data.members.filter((task) => {
+        if (task.username) {
+          clearUserlist();
+          return task.username
+            .toLowerCase()
+            .startsWith(searchInput.toLowerCase());
+        }
+      });
+      if (searchInput != '' && !matches.length) {
+        const userName = document.createElement('small');
+        userName.classList.add('user-list');
+        clearUserlist();
+        userName.innerText = 'User not found';
+        const listItems = document.getElementById('assigneeInput');
+        listItems.appendChild(userName);
+      }
+      assigneeHTML(matches);
+    }
+  } catch {
+    return;
+  }
+}
+
+function clearUserlist() {
+  const userList = document.querySelectorAll('.user-list');
+  userList.forEach((user) => {
+    user.remove();
+  });
+}
+
+function assigneeHTML(matches) {
+  const listItems = document.getElementById('list-items');
+  if (matches.length) {
+    const searchList = getUniqueAssignees(matches);
+    searchList.map((list) => {
+      const listItem = document.createElement('p');
+      listItem.classList.add('list-item');
+      listItem.style.cursor = 'pointer';
+      listItem.setAttribute('onclick', "displayAssignee('" + list + "')");
```

Comment:
can't we make use of ES6 Template literals ⬇️ 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
used template literal